docs-source/content/userguide/managing-domains/_index.md

```diff
@@ -86,6 +86,23 @@ The following features are **not** certified or supported in this release:
 
 For up-to-date information about the features of WebLogic Server that are supported in Kubernetes environments, see My Oracle Support Doc ID 2349228.1.
 
+### Meet Kubernetes resource name restrictions
+          
+Kubernetes requires the names of some resource types to follow the DNS label standard as defined in [DNS Label Names](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-label-names) and [RFC 1123](https://tools.ietf.org/html/rfc1123). This requirement restricts the characters that are allowed in the names of these resources, and also limits the length of these names to no more than 63 characters.
+
+The following is a list of such Kubernetes resources that the operator generates when a domain resource is deployed, including how their names are constructed.
+          
+* A domain introspector job named as `<domainUID>-<introspectorJobNameSuffix>`. The default suffix is `-introspector`, which can be overridden using the operator's Helm configuration `introspectorJobNameSuffix` (see [WebLogic domain Management]({{< relref "/userguide/managing-operators/using-the-operator/using-helm#weblogic-domain-management" >}})).
```

Comment:
named as -> named (globally; I understand that you are illustrating a naming pattern, but it is acceptable to say that it is named such and such versus named **as** such and such.)
WebLogic domain Management -> WebLogic domain management